packages/reparse-unix/reparse-unix.2.0.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "Provides support for parsing files as source of input for reparse library "
+description:
+  "Provides support for parsing files as source of input for reparse library "
+maintainer: ["Bikal Lem"]
+authors: ["Bikal Lem <gbikal@gmail.com>"]
+license: "MPL-2.0"
+homepage: "https://github.com/lemaetech/reparse"
+bug-reports: "https://github.com/lemaetech/reparse/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.10.0"}
+  "reparse"
```

Comment:
```suggestion
  "reparse" {=version}
```
We tend to strictly tie packages that are released from the same repository. This usually simplifies tagging releases and prevents issues when new releases are published with api changes